src/PhpSpec/Formatter/Presenter/StringPresenter.php

```diff
@@ -345,7 +345,8 @@ private function shouldSkipTracePresentation(array $call)
     private function presentCallable($value)
     {
         if (is_array($value)) {
```

Comment:
@ciaranmcnulty presentCallable is called only with a valid callable. Callable arrays have always a length of 2